### ISSUE TITLE
New image for target objectives

### ISSUE THREAD

**Issue by TuanDQ-Tuev-Sued** [EclipseFdn/openpass.eclipse.org#96]
I want to have the image (attached) as a new image for the target objectives https://openpass.eclipse.org/target_objectives/. 

![target-objectives-screenshot](https://user-images.githubusercontent.com/54616555/102595738-6259b980-4118-11eb-8325-fb3cf6bf5582.jpg)

**Comment by flora8984461** [comment]
Hi @TuanDQ-Tuev-Sued , the  three blocks `openPASS (open Platform for Assessment of Safety Systems)`, `High level of transparency and acceptance through publicly available open source platform.` and `Harmonized and flexible platform for effectiveness assessment of advanced driver assistance systems and automated driving.` are html css code for better SEO and accessibility.

Please check the preview: 

If this looks repeating, I can reproduce the styles as your new blocks in the image, and if possible, would you mind providing me an image with only the middle part? 

Thanks!

**Comment by chrisguindon** [comment]
@flora8984461 I think it would be best to crop those sections from the image and avoid duplication if @TuanDQ-Tuev-Sued is ok with that!

**Comment by flora8984461** [comment]
> @flora8984461 I think it would be best to crop those sections from the image and avoid duplication if @TuanDQ-Tuev-Sued is ok with that!

Sounds good. I'm going to crop those sections from the image and avoid duplication.  I'll make the two blocks on the top the same style as in the img.

**Comment by TuanDQ-Tuev-Sued** [comment]
> > @flora8984461 I think it would be best to crop those sections from the image and avoid duplication if @TuanDQ-Tuev-Sued is ok with that!
> 
> Sounds good. I'm going to crop those sections from the image and avoid duplication. I'll make the two blocks on the top the same style as in the img.

@flora8984461 Yes, if you can replicate exactly the same style of the images that would be great! So the new blocks should have the same colour and rounded corners. Our goal is to have a **consistent style** on the whole webpage. If you look at the images under the tab "Architecture" all the images have the same style. It would be great if you could do it exactly for the the "Target Objectives". Attached you find the middle part of the image.
![target-objectives](https://user-images.githubusercontent.com/54616555/102755186-58c98f00-436e-11eb-9f9c-c6a726b1d1d1.jpg)

**Comment by flora8984461** [comment]
@TuanDQ-Tuev-Sued  Thanks for your explanation.
I have updated the page and please check the preview here: 
https://deploy-preview-97--eclipsefdn-openpass.netlify.app/target_objectives/

Please let me know if this looks good or other changes are needed?

Thanks!

-------------------------------

> > > @flora8984461 I think it would be best to crop those sections from the image and avoid duplication if @TuanDQ-Tuev-Sued is ok with that!
> > 
> > 
> > Sounds good. I'm going to crop those sections from the image and avoid duplication. I'll make the two blocks on the top the same style as in the img.
> 
> @flora8984461 Yes, if you can replicate exactly the same style of the images that would be great! So the new blocks should have the same colour and rounded corners. Our goal is to have a **consistent style** on the whole webpage. If you look at the images under the tab "Architecture" all the images have the same style. It would be great if you could do it exactly for the the "Target Objectives". Attached you find the middle part of the image.
> ![target-objectives](https://user-images.githubusercontent.com/54616555/102755186-58c98f00-436e-11eb-9f9c-c6a726b1d1d1.jpg)

**Comment by TuanDQ-Tuev-Sued** [comment]
> @TuanDQ-Tuev-Sued Thanks for your explanation.
> I have updated the page and please check the preview here:
> https://deploy-preview-97--eclipsefdn-openpass.netlify.app/target_objectives/
> 
> Please let me know if this looks good or other changes are needed?
> 
> Thanks!
> 
> > > > @flora8984461 I think it would be best to crop those sections from the image and avoid duplication if @TuanDQ-Tuev-Sued is ok with that!
> > > 
> > > 
> > > Sounds good. I'm going to crop those sections from the image and avoid duplication. I'll make the two blocks on the top the same style as in the img.
> > 
> > 
> > @flora8984461 Yes, if you can replicate exactly the same style of the images that would be great! So the new blocks should have the same colour and rounded corners. Our goal is to have a **consistent style** on the whole webpage. If you look at the images under the tab "Architecture" all the images have the same style. It would be great if you could do it exactly for the the "Target Objectives". Attached you find the middle part of the image.
> > ![target-objectives](https://user-images.githubusercontent.com/54616555/102755186-58c98f00-436e-11eb-9f9c-c6a726b1d1d1.jpg)

@flora8984461 This looks already great! Can you please use a bold font for "Harmonized and flexible platform for ..." Furthermore, can you please make the font for alle 3 boxes slidly larger? Because the fontsize in comparison to the picture is smaller, it should be at least the same size.

**Comment by flora8984461** [comment]
@TuanDQ-Tuev-Sued Thanks for your feedback.
I have increase the font size and make the text bold. Please check the preview here:
https://deploy-preview-97--eclipsefdn-openpass.netlify.app/target_objectives/

I also made the specific text (advanced driver assistance systems and automated driving) turn to the second line to make two lines have a similar length like in the image.

Please let me know if this looks good or other changes are needed?

Thanks!

**Comment by TuanDQ-Tuev-Sued** [comment]
> @TuanDQ-Tuev-Sued Thanks for your feedback.
> I have increase the font size and make the text bold. Please check the preview here:
> https://deploy-preview-97--eclipsefdn-openpass.netlify.app/target_objectives/
> 
> I also made the specific text (advanced driver assistance systems and automated driving) turn to the second line to make two lines have a similar length like in the image.
> 
> Please let me know if this looks good or other changes are needed?
> 
> Thanks!

@flora8984461 Thank you! This looks good! This issue is fixed.

**Comment by TuanDQ-Tuev-Sued** [comment]
@flora8984461 Unfortunately, the appearance looks like this:
![image](https://user-images.githubusercontent.com/54616555/102800799-a832ae00-43b4-11eb-8887-71f495015cf1.png)

Can you please fix that the boxes are shown correctly?

**Comment by flora8984461** [comment]
Hi @TuanDQ-Tuev-Sued , it works for me and my team member. I assume it's a caching issue.

Could you please try hard fresh on your browser? 
```
Google Chrome
Windows users: hold down Ctrl and then press F5 on your keyboard.
Mac users: hold down Cmd and Shift and then press R on your keyboard.
```

Or you can try to open the page in an anonymous browser to see if it still shows like this? 

Thanks!
